src/html5/input_stream.rs

```diff
@@ -50,58 +50,46 @@ impl fmt::Display for Position {
 /// a surrogate characters since these cannot be stored in a single char.
 /// Eof is denoted as a separate element.
 #[derive(PartialEq, Debug, Copy, Clone)]
-pub enum Element {
+pub enum Bytes {
     /// Standard UTF character
-    Utf8(char),
+    Ch(char),
     /// Surrogate character (since they cannot be stored in char)
     Surrogate(u16),
     /// End of stream
     Eof,
 }
```

Comment:
The name of this struct will change from `Bytes` to something else in a later PR, once we deal with surrogate characters at an earlier stage of the input stream.

Reply:
The name `Ch` was chosen (1) because it's short and (2) because the Rust name for a UTF-8 character is `char`.